dev/customer_testing/run_tests.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+import 'package:glob/glob.dart';
+
+Future<void> main(List<String> arguments) async {
+  exit(await run(arguments) ? 0 : 1);
+}
+
+Future<bool> run(List<String> arguments) async {
+  final ArgParser argParser = ArgParser(
+    allowTrailingOptions: false,
+    usageLineLength: 72,
+  )
+    ..addOption(
+      'repeat',
+      defaultsTo: '1',
+      help: 'How many times to run each test. Set to a high value to look for flakes.',
+    )
+    ..addFlag(
+      'skip-on-fetch-failure',
+      defaultsTo: false,
+      help: 'Whether to skip tests that we fail to download.',
+    )
+    ..addFlag(
+      'skip-template',
+      defaultsTo: false,
+      help: 'Whether to skip tests named "template.dart".',
+    )
+    ..addFlag(
+      'verbose',
+      defaultsTo: false,
+      help: 'Describe what is happening in detail.',
+    )
+    ..addFlag(
+      'help',
+      defaultsTo: false,
+      negatable: false,
+      help: 'Print this help message.',
+    );
+
+  final ArgResults parsedArguments = argParser.parse(arguments);
+
+  final int repeat = int.tryParse(parsedArguments['repeat']);
+  final bool skipOnFetchFailure = parsedArguments['skip-on-fetch-failure'];
+  final bool skipTemplate = parsedArguments['skip-template'];
+  final bool verbose = parsedArguments['verbose'];
+  final bool help = parsedArguments['help'];
+  final List<File> files = parsedArguments
+    .rest
+    .expand((String path) => Glob(path).listSync())
+    .whereType<File>()
+    .where((File file) => !skipTemplate || path.basename(file.path) != 'template.test')
+    .toList();
+
+  if (help || repeat == null || files.isEmpty) {
+    print('run_tests.dart [options...] path/to/file1.test path/to/file2.test...');
+    print('For details on the test registry format, see:');
+    print('  https://github.com/flutter/tests/blob/master/registry/template.test');
+    print('');
+    print(argParser.usage);
+    print('');
+    return true;
```

Comment:
Nit: if `repeat == null`, I think false would be a more appropriate return value.

Reply:
replaced with `return help`.